plugins/oidc-auth/frontend/views/shared/_header_global.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="container global-header">
+  <nav class="navbar">
+    <div class="container">
+
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".nav-global-collapse">
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+      </div>
+
+      <% if !session[:user] %>
+        <button type="button" class="btn btn-default pull-right btn-collapse-login navbar-btn" data-toggle="collapse" data-target=".nav-global-collapse">
+          <%= I18n.t("navbar.login") %>
+        </button>
+      <% end %>
+      <div class="navbar-collapse nav-global-collapse collapse navbar-right navbar-default">
+        <ul class="nav pull-right navbar-nav navbar-login">
+          <%= render "shared/header_user" %>
+          <!-- PLUGIN BEGIN -->
+          <% if AppConfig[:oidc_definition] and !session[:user] %>
+            <li>
+              <%= link_to AppConfig[:oidc_definition][:label], "#{AppConfig[:frontend_proxy_prefix]}auth/#{AppConfig[:oidc_definition][:provider]}", method: :post %>
+            </li>
+          <% end %>
+          <!-- PLUGIN END -->
+          <% if ArchivesSpaceHelp.enabled? %>
+            <li><%= link_to_help :link_opts => {"data-placement" => (session[:user] ? "left" : "bottom")} %></li>
+          <% end %>
+        </ul>
+      </div><!-- nav-collapse -->
+    </div>
+  </nav>
+</div>
+<!-- PLUGIN BEGIN -->
+<% if session[:user] %>
+  <% if AppConfig[:oidc_definition] %>
+    <script type="text/javascript">
+      const addOIDCLogout = () => {
+        const logoutURL = "<%= AppConfig[:frontend_proxy_prefix] %>" + "logout";
+        const logout = document.querySelector(`li>a[href~="${logoutURL}"]`);
+        if (logout === null) {
+          console.error(`No logout URL found; searched for ${logoutURL}`)
+          return
+        }
+
+        const newLogoutLink = document.createElement('a');
+        newLogoutLink.setAttribute('href', "<%= AppConfig[:frontend_proxy_prefix] %>" + "auth/logout");
+        newLogoutLink.innerText = "<%= AppConfig[:oidc_definition][:label] %>" + " Logout";
+        const newLogoutItem = document.createElement('li');
+        newLogoutItem.append(newLogoutLink);
+        logout.parentNode.parentNode.append(newLogoutItem);
+      };
+      addOIDCLogout();
+    </script>
+  <% end %>
```

Comment:
IF we want to remove the default logout, we may be able to override the current logout.